b9lab-content/5-my-own-chain/03-starport-03-stored-game.md

```diff
@@ -0,0 +1,204 @@
+---
+title: The Stored Game Object
+order: 4
```

Comment:
Are there a few articles before the tutorial? 

There might be missing the 

`starport scaffold chain github.com/username/checkers`

command to get started, maybe some other commands that I am missing before this chapter?

Reply:
Added a couple paragraphs as a glue.

Reply:
@xavierlepretre is this feedback implemented completely? If yes, please resolve this conversation.